code/modules/economy/holopay.dm

```diff
@@ -173,32 +203,40 @@
 	set_light(2)
 	visible_message(span_notice("A holographic pay stand appears."))
 	/// Start checking if the source projection is in range
-	RegisterSignal(card, COMSIG_MOVABLE_MOVED, .proc/check_operation)
-	if(card.loc)
-		holder = WEAKREF(card.loc)
-		RegisterSignal(card.loc, COMSIG_MOVABLE_MOVED, .proc/check_operation)
+	RegisterSignal(linked_card, COMSIG_MOVABLE_MOVED, .proc/handle_move)
```

Comment:
you'll want to call track on linked_card here, since otherwise anything it spawns in won't be properly tracked

Reply:
I don't think it's possible to project with it when it's inside something because you need to call attack_self_secondary() or attack_secondary() with it but I've added it assuming you know something I don't

Reply:
Sitting in someone's hand counts as "in" them is the trouble

Reply:
Of course, what a moron